doc/source/whatsnew/v0.16.0.txt

```diff
@@ -139,6 +139,8 @@ Bug Fixes
   SQLAlchemy type  (:issue:`9083`).
 
 
+- Items in ``Categorical.unique()`` now appear in the order in which are originally found, not in sorted order (:issue:`9331`). This is consistent with the behavior for other dtypes.
+
```

Comment:
Items in ``Categorical.unique()`` (and ``s.unique()`` if ``s`` is of dtype ``category``) now appear in the order in which _they_ "(?) are originally found, not in sorted order (:issue:`9331`). Formerly it was modeled after `np.unique()` which does sort the values but is now consistent with the behavior of ``unique()`` for other pandas dtypes.

Not sure about the "they"...